modules/show_recent_added.js

```diff
@@ -0,0 +1,48 @@
+let recentChecked;
+
+// Scroll to show recent added books and highlights them when clicking link1
+function scrollShowRecentAdded(allRecent = document.querySelectorAll('.recent-added')) {
+  if (allRecent[0]) {
+    setTimeout(() => { allRecent[0].scrollIntoView(); }, 900);
+
+    setTimeout(() => {
+      allRecent.forEach((recent) => {
+        recent.classList.toggle('highlight-recent');
+      });
+    }, 1200);
+
+    setTimeout(() => {
+      allRecent.forEach((recent) => {
+        recent.classList.toggle('highlight-recent');
+      });
+    }, 1500);
+
+    recentChecked = true;
+  }
+}
+
+// Remove recent added style when clicking link2 or link2
+function removeRecentAddedStyle(allRecent = document.querySelectorAll('.recent-added')) {
+  if (allRecent.length > 0) {
+    allRecent.forEach((recent) => {
+      recent.classList.remove('recent-added');
+    });
+    recentChecked = false;
+  }
+}
+
+// Checks window.location.hash and executes either of functions above
+export default function showRecentAdded(
```

Comment:
- [ ] It would be better to use arrow functions here too, as this is a better practice in ES6, please do this for all the functions.